test/src/test-reader/test-parser.js

```diff
@@ -684,6 +684,10 @@ describe("test-reader/test-parser", () => {
                             ["smoke", false],
                             ["slow", false],
                         ]),
+                        parent: {
+                            fullTitle: () => "Some parent",
+                            tags: new Map([]),
```

Comment:
Shouldn't it be `Set`?

Also, as far as I can tell, the bug was that if parent has tag "smoke" and we ran testplane with `!smoke`, the test would still launch, because it was enough to get ok only from children. But here you just add an empty array, is this really sufficient to test the case we uncovered now?